### ISSUE TITLE
[Request] Colored logging

### ISSUE THREAD

**Issue by Python1320** [lunarmodules/lualogging#10]
Adding colors makes console output instantly more readable. Unfortunately it is not very well supported and as far as I can see the most you can support it with the present piping here is by doing something like this:
```lua
log=require'logging.console'{logPattern="\x1b[31m%date\x1b[1m %level\x1b[0m %message\x1b[0m\n"}
log:info"test"
```
![image](https://user-images.githubusercontent.com/207340/121776108-c9ba6880-cb93-11eb-852e-adf9879bc9e3.png)

If someone has a better solution or suggestions, please share!

**Comment by Tieske** [comment]
this is hard to add because it is platform depedent

**Comment by Tieske** [comment]
just released 1.6.0 which allows for message formats per log-level. If you inject ansi codes in there, you'll get colorization.

example: https://github.com/xHasKx/luamqtt/blob/3aaab293e9ac26fc8820bc38bf0d04de68b11517/tests/configure-log.lua

I'll close this now.
